yaml/csr.yaml.m4

```diff
@@ -4487,22 +4487,6 @@ ifelse(eval(CLIC != 0), 1, [[[
       lsb: 0
 ]]])
```

Comment:
Not okay to remove this from the yaml file as we try to have the same yaml between 40x and 40s. The CSR should be excluded for non-CLIC and if user mode is not supported.

Only this file should get merged to the 40S, not the other files

Reply:
Added mscratchcsw back in and guarded with CLIC and UMODE.